### ISSUE TITLE
debugfs does not work for originators any more

### ISSUE THREAD

**Issue by mweinelt** [ffnord/mesh-announce#36]
starting batman-adv v2018.2 according to tackin

```pytb
Traceback (most recent call last):
  File "/opt/mesh-announce/gather.py", line 62, in gather_data
    value = _eval_snippet(path, env)
  File "/opt/mesh-announce/gather.py", line 44, in _eval_snippet
    ret = eval(code, real_env)
  File "/opt/mesh-announce/neighbours.d/batadv", line 12, in <module>
    open('/sys/kernel/debug/batman_adv/{}/originators'.format(batadv_dev)))
FileNotFoundError: [Errno 2] No such file or directory: '/sys/kernel/debug/batman_adv/dom3-bat/originators'
```

**Comment by Grotax** [comment]
This is caused by a change in batman-adv 2018.2+ debugfs is now disabled by [default](https://www.open-mesh.org/issues/320#change-1418).
A workaround for this is to build the module with:
```
CONFIG_BATMAN_ADV_DEBUGFS=y
```
But i think it would be better to patch mesh-announce to use some other method.

**Comment by rubo77** [comment]
Can you create a pr with the quick fix?

And leave this issue open to develop some other method later

**Comment by Grotax** [comment]
I think you misunderstood me.
If you build batman-adv for your kernel and want debugfs enabled (in >=2018.2) you have to build the kernel module with:
```
make CONFIG_BATMAN_ADV_DEBUGFS=y
```
Which will enable debugfs in the resulting build.
After `make install` and a reboot your mesh-announce will probably work just fine.
To fix this in mesh-announce a new method is needed. But I have no idea what the best approach would be.

**Comment by genofire** [comment]
Could this issue closed now? (Because #41 was merged)

**Comment by Grotax** [comment]
Yes, can be closed.
